wherehows-web/app/templates/components/purge-policy.hbs

```diff
@@ -9,40 +9,65 @@
 <ul class="purge-policy-list">
   {{#if isEditable}}
 
-    {{#each-in purgePolicyProps as |purgeType prop|}}
-      <li
-        class="purge-policy-list__item {{unless (contains (uppercase platform) prop.platforms)
-                                                'purge-policy-list__item--disabled'}}">
-
-        {{#radio-button-composer
-          name="dataset-purge-policy"
-          value=(if (contains (uppercase platform) prop.platforms) purgeType null)
-          disabled=(if (contains (uppercase platform) prop.platforms) false true)
-          groupValue=(readonly purgePolicy)
-          changed=(action "onChange")
-        }}
-          {{prop.displayAs}}
-        {{/radio-button-composer}}
-
-        {{#unless (contains (uppercase platform) prop.platforms)}}
-          <p class="purge-policy-list__availability">
-            Purge policy not available for <span class="purge-policy-list__platform--unavailable">
-            {{lowercase platform}}</span>
-          </p>
-        {{/unless}}
-
-        <p>{{prop.desc}}</p>
-
-        {{#if (and requestExemptionReason (eq purgeType exemptPolicy))}}
-
-          {{medium-content-editable
-            value=purgeNote
-            options=editorOptions
-            class="comment-new__content"}}
-
-        {{/if}}
-      </li>
-    {{/each-in}}
+    {{#if getPlatformPolicies.isRunning}}
+      {{pendulum-ellipsis-animation}}
+    {{/if}}
+
+    {{#if getPlatformPolicies.last.isError}}
+    {{!--todo: swap out with error-state component--}}
+      {{empty-state
+        heading="Purge Policies not available"
+        subHead="Could not find supported purge policies for this platform"
+      }}
+
+      <div class="purge-policy-list__retry-platforms">
+        <button
+          class="nacho-button nacho-button--large-inverse"
+          onclick={{perform getPlatformPolicies}}>
+          Retry
+        </button>
+      </div>
+
+    {{/if}}
+
+    {{#if getPlatformPolicies.last.isSuccessful}}
+
+      {{#each-in purgePolicyProps as |purgeType prop|}}
+        <li
+          class="purge-policy-list__item {{unless (contains purgeType supportedPurgePolicies)
+                                                  'purge-policy-list__item--disabled'}}">
+
+          {{#radio-button-composer
+            name="dataset-purge-policy"
+            value=(if (contains purgeType supportedPurgePolicies) purgeType null)
+            disabled=(if (contains purgeType supportedPurgePolicies) false true)
+            groupValue=(readonly purgePolicy)
+            changed=(action "onChange")
```

Comment:
I saw a onPolicyChange defined above. Is this supposed to be that or "onChange" as is?

Reply:
`onPolicyChange` is  the external action passed in to the `PurgePolicy` component from the containing context.  the `PurgePolicy` component has an onChange handler that invokes that